_commands/basics/id.md

```diff
@@ -3,22 +3,58 @@
 
 id
 --
-TODO: Add documentation for this command by submitting a pull request.
+`id` is a command that prints out the effective and real user and group ids.
 <!-- one line explanation would go here -->
-
+The syntax for the command is:
 <!-- minimal example -->
+
 ~~~ bash
-*short* example of command usage and output
+id [option(s)]
+id [option(s)] [USERNAME]
 ~~~
 
+ * The `id` command prints out information for the current user. However, it is possible to specify a different user by appending their name to the end of the command.
```

Comment:
This shows up on the preview page `/reference` as a rather large amount of text. This can be moved as the first example with the comment above.